identity/webapp/next.config.js

```diff
@@ -0,0 +1,89 @@
+const webpack = require('webpack');
+const path = require('path');
+const withTM = require('next-transpile-modules')(['@weco']);
+const withBundleAnalyzer = require('@next/bundle-analyzer');
+const withMDX = require('@next/mdx')({
+  extension: /\.(md|mdx)$/,
+});
+const buildHash = process.env.BUILD_HASH || 'test';
+const isProd = process.env.NODE_ENV === 'production';
+
+const config = function (webpack) {
+  const prodSubdomain = process.env.PROD_SUBDOMAIN || '';
+  const withBundleAnalyzerConfig = withBundleAnalyzer({
+    analyzeServer: ['server', 'both'].includes(process.env.BUNDLE_ANALYZE),
+    analyzeBrowser: ['browser', 'both'].includes(process.env.BUNDLE_ANALYZE),
+    bundleAnalyzerConfig: {
+      server: {
+        analyzerMode: 'static',
+        generateStatsFile: true,
+        statsFilename: `../../.dist/server.${buildHash}.json`,
+        reportFilename: `../../.dist/server.${buildHash}.html`,
+        openAnalyzer: false,
+      },
+      browser: {
+        analyzerMode: 'static',
+        generateStatsFile: true,
+        statsFilename: `../.dist/browser.${buildHash}.json`,
+        reportFilename: `../.dist/browser.${buildHash}.html`,
+        openAnalyzer: false,
+      },
+    },
+    webpack(config, options) {
+      config.plugins.push(
+        new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+        new webpack.NormalModuleReplacementPlugin(
+          /moment-timezone\/data\/packed\/latest\.json/,
+          path.join(__dirname, 'timezones.json')
+        )
+      );
+
+      config.module.rules.push({
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+      });
+
+      return config;
+    },
+  });
+
+  const apmConfig = {
+    environment: process.env.APM_ENVIRONMENT,
+    serverUrl: process.env.APM_SERVER_URL,
+    centralConfig: true,
+  };
+
+  const rewrites =
```

Comment:
Think once we have integrated NextJS into a "custom server" using the existing koa stuff we'll be able to remove this